lib/libyui_client/wait.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module LibyuiClient
+  module Wait
+    def self.until(timeout:, interval:, message: nil, object: nil)
+      Timer.wait(timeout, interval) do
+        result = yield(object)
+        return result if result
+      end
+      raise Error::TimeoutError, "timed out after #{timeout} seconds.#{message}"
```

Comment:
Nitpicking. Let's add space after dot, so message is `. Message` and not `.Message`.

Reply:
Done.